app/views/admin/edition_tags/edit.html.erb

```diff
@@ -1,76 +1,83 @@
-    <p><a href="https://www.gov.uk/guidance/how-to-publish-on-gov-uk/creating-and-updating-pages#add-topic-tagging">Find out more about tagging to the topic taxonomy.</a></p>
-    <hr>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <div class="govuk-!-margin-bottom-7">
+      <%= render "govuk_publishing_components/components/notice", {
+        title: "Topic taxonomy tags are applied to all editions of a document",
+        description_govspeak: sanitize("<p>This means changes will be reflected on the live site if there's a published edition.</p>"),
+        show_banner_title: true,
+        banner_title: "Information"
+      } %>
 
-    <%= form_for @tag_form, url: admin_edition_tags_path(@edition), method: :put do |form| %>
-      <%= form.hidden_field :previous_version %>
+      <p class="govuk-body">The topic taxonomy groups content based on what it’s about.</p>
+      <p class="govuk-body">Choose the tag or tags that best describe what this content is about.</p>
+      <p class="govuk-body">You can use the whole topic taxonomy. There's no limit to the number of tags you can choose.</p>
+      <p class="govuk-body">
+        <a class="govuk-link"
+          href="https://www.gov.uk/guidance/how-to-publish-on-gov-uk/creating-and-updating-pages#add-topic-tagging">
+          Find out more about tagging to the topic taxonomy.
+        </a>
+      </p>
+    </div>
 
-      <div class="form-group"
-           data-module="taxonomy-tree-checkboxes"
-           data-content-id="<%= @edition.content_id %>"
-           data-content-format="<%= @edition.content_store_document_type %>"
-           data-content-public-path="<%= public_document_path(@edition) %>">
+    <%= render "govuk_publishing_components/components/heading", {
+      text: "Topic taxonomy tags",
+      margin_bottom: 2,
+    } %>
 
-        <%= render partial: "/admin/shared/tagging/taxonomy", locals: { selected_taxons: @tag_form.selected_taxons, level_one_taxons: @topic_taxonomy.ordered_taxons } %>
+    <%= form_for @tag_form, url: admin_edition_tags_path(@edition), method: :put do |form| %>
+      <%= form.hidden_field :previous_version %>
 
-      </div>
+      <% if @topic_taxonomy.ordered_taxons_transformed(@tag_form.selected_taxons).count() %>
+        <%= render partial: "/components/miller-columns", locals: {
+          id: "taxonomy_tag_form[taxons]",
+          items: @topic_taxonomy.ordered_taxons_transformed(@tag_form.selected_taxons),
+        } %>
+      <% else %>
+        <p class="govuk-body">No taxons available</p>
+      <% end %>
 
       <%= form.hidden_field "invisible_taxons", value: @tag_form.invisible_taxons.join(",") %>
 
-      <p>
+      <p class="govuk-body">
         <%= link_to "Suggest a new tag",
           "#{Whitehall.support_url}/taxonomy_new_topic_request/new",
-          class: "feedback-link"
+          class: "feedback-link govuk-link"
```

Comment:
Can we can remove the class `feedback-link` now - it doesn't seem to do anything?

Reply:
done